### PR TITLE
Tortilla

### PR DESCRIPTION
**Pull request by DIvkov575**
### Describe your change:

* [x] Add an algorithm?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: physics/Torricelli.py
Comment (ZeroDayOwl) [inline physics/Torricelli.py:8]:
```suggestion
- https://en.wikipedia.org/wiki/Torricelli%27s_law
```

File: physics/Torricelli.py
Comment (ZeroDayOwl) [inline physics/Torricelli.py:5]:
```suggestion
Velocity = sqrt(2 * Acceleration due to gravity * Height)
```

File: physics/Torricelli.py
Comment (ZeroDayOwl) [inline physics/Torricelli.py:21]:
```suggestion
        gravity: Acceleration due to gravity. Gravitational force on system,
```

File: physics/Torricelli.py
Comment (ZeroDayOwl) [inline physics/Torricelli.py:20]: Please mention the unit of height used.

File: physics/Torricelli.py
Comment (ZeroDayOwl) [inline physics/Torricelli.py:32]:
```suggestion
        raise ValueError("The value of g cannot be equal or smaller than 0.")
```

File: physics/archimedes_principle.py
Comment (ZeroDayOwl) [inline physics/archimedes_principle.py:29]: If the default value of g is 9.8, you can add a new test case for value of g different than 9.8; otherwise, it seems redundant.

File: physics/archimedes_principle.py
Comment (ZeroDayOwl) [inline physics/archimedes_principle.py:36]:
```suggestion
        raise ValueError("Fluid density cannot be equal or smaller than 0.")
```

File: physics/archimedes_principle.py
Comment (ZeroDayOwl) [inline physics/archimedes_principle.py:40]:
```suggestion
        raise ValueError("Object volume cannot smaller than 0")
    if gravity <= 0:
        raise ValueError("Acceleration due to gravity cannot be equal or smaller than 0.")
```

**Review by ZeroDayOwl — CHANGES_REQUESTED** [review]

**Comment by ZeroDayOwl** [comment]
@DIvkov575 make one PR per algorithm. You have duplications present in #7143 and #7200

**Comment by cclauss** [comment]
Hacktoberfest is strenuous for our team of active volunteer maintainers. That small team now has ___200+ open pull requests___ to review.  We have __high standards for our submissions__ which are described in [`CONTRIBUTING.md`](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md#contributing-guidelines).  Our continuous integration tests help to ensure that our standards are met before we review and merge deserving pull requests.  This is further complicated by the fact that we already have [lots of algorithms](https://github.com/TheAlgorithms/Python/blob/master/DIRECTORY.md) and would like to avoid duplicates.

For these reasons, I will **close all open pull requests failing tests** so that our unpaid team has a smaller mountain to climb.  If you want us to land your pull request, please read [`CONTRIBUTING.md`](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md#contributing-guidelines) _carefully_ and try again.

**Pull request closed without merging**
